AppStoreExample/AppStoreExample/ViewModels/AppSyncViewModel.swift

```diff
@@ -8,43 +8,51 @@
 
 import Foundation
 
+enum AppSyncActions {
```

Comment:
This should be an embedded enum inside AppSyncViewModel and should be named SyncStatus